tools/config/Tomb1Main_ConfigTool/Resources/Lang/en.json

```diff
@@ -239,6 +239,10 @@
       "Title": "Change Pierre spawn behavior",
       "Description": "Makes a freshly triggered (runaway) Pierre replace an already existing (runaway) Pierre."
     },
+    "restore_ps1_enemies": {
+      "Title": "Restore PS1 enemies",
+      "Description": "Adds the mummy that appears in the PlayStation version of City of Khamoon, room 25.\nNB Changing this option will require restarting the level."
```

Comment:
What does `NB` mean?

Reply:
Nota bene - maybe too formal?

Reply:
Let's just remove it, as it doesn't add much value

Reply:
NB: NB removed :)